### ISSUE TITLE
Uncaught TypeError: Cannot read property 'id' of null

### ISSUE THREAD

**Issue by cryptix** [dominictarr/push-mux#1]
I'm getting this with error from push-mux@1.0.3 with the current dev version of [talenet](https://github.com/talenet/talenet).

I tried to get all deps updated to muxrpc@7, here is our current [npm ls.txt](https://github.com/push-stream/push-mux/files/1725431/npm_ls.txt). 

```
/home/user/TALEnet/node_modules/push-mux/index.js:217 Uncaught TypeError: Cannot read property 'id' of null
    at Mux._credit (/home/user/TALEnet/node_modules/push-mux/index.js:217)
    at Sub._preread (/home/user/TALEnet/node_modules/push-mux/sub.js:54)
    at Sub.DuplexStream._write (/home/user/TALEnet/node_modules/push-mux/stream.js:31)
    at writeDataToStream (/home/user/TALEnet/node_modules/push-mux/index.js:83)
    at Mux.write (/home/user/TALEnet/node_modules/push-mux/index.js:161)
    at next (/home/user/TALEnet/node_modules/push-stream-to-pull-stream/sink.js:30)
    at /home/user/TALEnet/node_modules/pull-goodbye/node_modules/pull-stream/throughs/filter.js:17
    at Object.cb (/home/user/TALEnet/node_modules/packet-stream-codec/index.js:111)
    at drain (/home/user/TALEnet/node_modules/pull-reader/index.js:39)
    at more (/home/user/TALEnet/node_modules/pull-reader/index.js:51)
```

**Comment by dominictarr** [comment]
thanks, fixed in 1.0.4

**Comment by dominictarr** [comment]
this should be fixed in push-mux@1.0.4

**Comment by cryptix** [comment]
Thanks! Tried and didn't appear yet!
